### PR TITLE
Improve SQL Server requirements

### PR DESCRIPTION
**Pull request by mosabua**
Consistent with PostgreSQL now

### CODE REVIEW

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 1]
```diff
@@ -10,8 +10,12 @@ between two different SQL Server instances.
 Requirements
 ------------
 
-The connector supports SQL Server 2016, SQL Server 2014, SQL Server 2012
-and Azure SQL Database as connected data sources.
+Requirements for using the connector in a catalog to connect to a PostgreSQL
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/sqlserver.rst:13]:
```suggestion
Requirements for using the connector in a catalog to connect to a SQL Server
```

**Review by hashhar — APPROVED** [review]
LGTM % comment.

File: docs/src/main/sphinx/connector/sqlserver.rst (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/sqlserver.rst]: Fixed

**Review by Ordinant — APPROVED** [review]
Ship it.

File: docs/src/main/sphinx/connector/sqlserver.rst [hunk 2]
```diff
@@ -10,8 +10,12 @@ between two different SQL Server instances.
 Requirements
 ------------
 
-The connector supports SQL Server 2016, SQL Server 2014, SQL Server 2012
-and Azure SQL Database as connected data sources.
+Requirements for using the connector in a catalog to connect to a SQL Server
```
Comment (m57lyra) [inline docs/src/main/sphinx/connector/sqlserver.rst:13]: GDDSG recommends active voice whenever possible: https://developers.google.com/tech-writing/one/active-voice

**Review by m57lyra — APPROVED** [review]

**Review by electrum — APPROVED** [review]

**Pull request merged**